src/style.css

```diff
@@ -1 +1,64 @@
-/* add styles here */
+
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: Arial, sans-serif;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star:not(:last-child) {
+  margin-right: 4px;
+}
+
+.stars--0 .stars__star,
+.stars__star:nth-child(2),
+.stars__star:nth-child(3),
+.stars__star:nth-child(4),
+.stars__star:nth-child(5) {
+  background-image: url('images/star.svg');
+}
```

Comment:
don't repeat all possible child elements instead use formula (an + b) 
you can read more about it [here](https://css-tricks.com/how-nth-child-works/)

Reply:
still to fix :wink: